src/app/tests/suites/certification/Test_TC_TSUIC_2_2.yaml

```diff
@@ -35,9 +35,13 @@ tests:
       arguments:
           value: 0
 
-    #Disabled as UI prompts required for user action
     - label: "Verify device temperature displayed in °C"
-      disabled: true
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      arguments:
+          values:
+              - name: "message"
+                value: "Verify device temperature displayed in °"
```

Comment:
Should be
                value: "Verify device temperature displayed in °**C**"

Reply:
i will update it thanks.